src/PlantBundle/Controller/PlantController.php

```diff
@@ -30,7 +42,51 @@ public function showAction($id)
 
         $plant= $em->getRepository('PlantBundle:Plant')->find($id);
 
+        if ($plant==null)
```

Comment:
Do not check for null with `==`. Use `===` (strict comparision).

Look below to learn more:
```
0==false     <- this is true
0===false    <- this is false
null==false  <- this is true
null===false <- this is false
0==null      <- this is true
```